### DISCUSSION TITLE
Suddenly can't add GPG keys anymore

### DISCUSSION THREAD

**Discussion by filefqa** [community/community#62536] [New to GitHub] [answered]
I have successfully added and removed about 5 GPG signing keys today. This was probably not the best idea, but I was trying out different configurations of my key pair and had to create and upload those new keys to Github to verify that they worked (and then delete them).

In the end, I figured I will use the ed448 algorithm but when I tried uploading **that** key, I got the following error message:

![изображение](https://github.com/community/community/assets/141078469/bd0d7b46-d3f5-441c-ada7-8c1c27260dfd)

I searched for solution to this issue and the author of [this discussion](https://github.com/orgs/community/discussions/45937) also had similar problems adding ed448 key. At this point I thought that I should fall back to using RSA4096 which worked just fine for me in previous keys, but adding literally **any** GPG key now results in the same error being shown.

I followed the [guide](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification) in all my attempts and as I previously said, it worked for me before. I can also still add SSH keys without any problem.

My first instinct is that perhaps I tampered with my keys too much in a short period of time and therefore my account is now kind of "rate limited", but unfortunately the error message is too vague to say for sure. Any help in identifying the problem is highly appreciated.

**Comment by queenofcorgis** [comment]
Hi there - are you still experiencing this? Also, I know this may be a bit simple, but to refresh the system - you may try logging out then back in.

**Reply by filefqa** [reply, marked as answer]
Sorry for the late reply, but yes, after 4 days of trying, it seems like I have found the solution. I won't try to verify this on other kinds of keys because I am still scared that it might stop working again, but what I did was:

1. Created the normal RSA4096 key (not the weird ed448 or something like that)
2. Did not create any subkeys
3. Imported the public key to the file instead of using command line and copied it from there

I am not sure if it's only one of these steps that caused the issue to go away, all of them, or something else entirely, but I have a working GPG key now at least.

**Reply by 4310V343k** [reply]
for me it was how different emulators copy stuff differently
![image](https://github.com/community/community/assets/56449763/187b9f36-0e56-4bc9-96a4-220477b7e644)
